<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSettingsWidget</class>
 <widget class="QWidget" name="NetworkSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="connect">
         <property name="text">
          <string>Connect to...</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/transmit_add.png</normaloff>:/icons/transmit_add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Manage...</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/transmit_edit.png</normaloff>:/icons/transmit_edit.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="turnOn">
         <property name="text">
          <string>Turn On</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/lightbulb.png</normaloff>:/icons/lightbulb.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="turnOff">
         <property name="text">
          <string>Turn Off</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/lightbulb_off.png</normaloff>:/icons/lightbulb_off.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Network Information</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="stateLabel">
            <property name="text">
             <string>Wifi is currently:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="state">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>ON</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="ssidLabel">
            <property name="text">
             <string>SSID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ssid">
            <property name="text">
             <string>kovanet</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="securityLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Security:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="security">
            <property name="text">
             <string>WPA2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
